src/solvers/dg/3d/dg.jl

```diff
@@ -1812,8 +1811,13 @@ function prolong2mortars!(dg::Dg3D, mortar_type::Val{:l2}, thread_cache)
   end
 end
 
-@inline function interpolate_mortar_values!(dg::Dg3D, ::Val{:l2}, leftright, m,
-                                            u_large, fstar_tmp1)
+"""
+    interpolate_mortar_values!(dg::Dg3D, ::Val{:l2}, leftright, m, u_large, fstar_tmp1)
+
+Interpolate `u_large` to `dg.l2mortars.u_[upper/lower]_[left/right]` for mortar `m`
+using the forward mortar operators of `dg` and `fstar_tmp1` as temporary storage.
+"""
+@inline function interpolate_mortar_values!(dg::Dg3D, ::Val{:l2}, leftright, m, u_large, fstar_tmp1)
```

Comment:
I was just thinking... would it maybe be nice to rename `interpolate_mortar_values` and `copy_and_project_mortar_fluxes` symmetrically to indicate their purpose instead of what exactly they are doing? E.g., something like
```
interpolate_mortar_values      -> element2mortar_solution
copy_and_project_mortar_fluxes -> mortar2element_fluxes
```
or
```
interpolate_mortar_values      -> element_solutions_to_mortars
copy_and_project_mortar_fluxes -> mortar_fluxes_to_elements
```
? I know, I know, the names are not perfect, but I'd think that having this symmetry in the name would maybe be more descriptive than the original names (where is something interpolated? are the fluxes first copied and then projected?) Just an idea and open for discussion though.

Reply:
I like that suggestion! Implemented in b1afd2b456d5e9b01e2f18d0c4ded31261568eb3